### ISSUE TITLE
[V1.1] Model Adapter Code Refinement

### ISSUE THREAD

**Issue by XinyaoWa** [intel/e2eAIOK#188]
- [x] Refine requirements.txt and can install in vanilla env
- [x] remove optuna  
- [x] remove useless code (draft / model_bak)
- [x] disable distiller save logits in distribution mode
- [x] bug fix
    - [x] Distiller save logtis
        - [x] can't close readfile 
        - [x] UT can't close 
        - [x] can't save logits after 1 epoch
    - [x] Trainer will fail to show parameter in tensorboard if directly call evaluate function

**Comment by xuechendi** [comment]
#192
